src/app/archive/archive_lib/processor.ml

```diff
@@ -1185,11 +1189,12 @@ module Block = struct
                       (id, secondary_sequence_no, fee_transfer.receiver_pk)
                       :: acc )
                 in
-                let fee_transfer_infos =
+                (* the fold reverses the order of the infos from the fee transfers *)
```

Comment:
Can you move this comment down to the `[id2; id1]` case? It'd be good to have the comment as close to the unexpected code as possible.

Reply:
Done.